src/main/resources/story/shop_api/GetCategoriesID.story

```diff
@@ -0,0 +1,77 @@
+Story: GET /categories/{id} test
+
+Scenario: check response body
+When I issue a HTTP GET request for a resource with the URL 'https://api.predic8.de/shop/categories/<category_id>'
+Then the response code is equal to '200'
+When I save response body to the scenario variable 'body'
+Then a JSON element from '${body}' by the JSON path '$' is equal to <expected response>
+Examples:
```

Comment:
How can I implement DDT approach here? I'd like to perform GET request for several <category_id> and for every category check that actual json matches expected.
It looks like I can't store json inside 'Examples'

Reply:
https://git.epam.com/valery_yatsynovich/vividus/wikis/Handbook/Expressions/Resources 
in theory you can hide json in txt file, but usually we don't check the whole json, just some elements and they can be reached by json path

Reply:
You can also use this step 'Then a JSON element by the JSON path '$jsonPath' is equal to '$expectedData'$options'  and here you can skip step6 as this step will refer to response json from the previous 'When I send...' step